baybe/recommenders/pure/bayesian/sequential_greedy.py

```diff
@@ -178,7 +178,7 @@ def _recommend_hybrid(
             # TODO: Currently assumes that discrete parameters are first and continuous
             #   second. Once parameter redesign [11611] is completed, we might adjust
             #   this.
-            candidates_comp.columns = list(range(len(candidates_comp.columns)))
+            candidates_comp.columns = list(range(len(candidates_comp.columns)))  # type: ignore[assignment]
```

Comment:
Just to understand: MyPy does not want you to explicitly re-assign a value to the already existing columns here? Or does it complain that you assign a `list` object to something that is technically not a list?

Reply:
this is part of the pandas mypy problems: mypy tells me thatonly `Index[str]` should be assigned here, but we have `list[int]`. In reality this is totally fine and I think even fully allowed
But the typing with pandas is not very good, in particular in such assignments (eg also with `.loc` or so) so we need to ignore (I also tried converting to `Index[str]` but ironically that breaks stuff)